specification/programmableconnectivity/ProgrammableConnectivity.Management/OperatorApiConnection.tsp

```diff
@@ -44,18 +44,28 @@ model OperatorApiConnectionProperties {
   /**
```

Comment:
I'm not sure about this. If an Operator decides to update their plans, we may want customers to be able to update this. I don't know what the process would be if that happens, and I don't think we should determine the process now, but I'm wondering if we should keep it open so that we can update this in the future if we need to (and in the meantime prevent updating it in our RP).